src/modules/organisation/api/organisation.controller.ts

```diff
@@ -38,4 +40,22 @@ export class OrganisationController {
         const createdOrganisation: CreatedOrganisationDto = await this.uc.createOrganisation(organisationDto);
         return this.mapper.map(createdOrganisation, CreatedOrganisationDto, OrganisationResponse);
     }
+
+    @Get(':organisationId')
+    @ApiCreatedResponse({ description: 'The organization was successfully pulled.' })
```

Comment:
This should be an ApiOkResponse and the response type is not shown on the OpenAPI documentation.